beets/ui/__init__.py

```diff
@@ -416,6 +416,19 @@ def human_seconds_short(interval):
     return u'%i:%02i' % (interval // 60, interval % 60)
 
 
+def raw_seconds_short(string):
+    """Formats a human-readable M:SS string as a float (number of seconds).
+
+    Raises ValueError if the conversion cannot take place due to `string` not
+    being in the right format.
+    """
+    match = re.match('^(\d+):([0-5]\d)$', string)
```

Comment:
A note about using a regexp: I have decided to go through this route to make it consistent with `human_seconds_short` (ie. format should be `(int)M:(int)SS`, where SS must be < 60). Using the `datetime` facilities would restrict M to be < 60.

It might be desirable to make it a bit more flexible (allow HH:MM:SS, or M:SS.zzzzz, etc), and it should not be too hard, but I've opted not to include those for keeping it as an "inverse" of sorts of `raw_seconds_short`.